core/src/main/scala/zio/logging/LogLevel.scala

```diff
@@ -16,7 +16,8 @@ sealed trait LogLevel { self =>
 
   def min(that: LogLevel): LogLevel = if (self > that) that else self
 }
-object LogLevel {
+object LogLevel       {
```

Comment:
@mijicd do you know how to get rid of this?

Reply:
Yes, you need to wrap it like this:

```scala
// format: off
object Dummy {
// format: on
```

Reply:
this is disgusting :(

Reply:
I'm not sure what's uglier, the formatting or these "magic" comments.

Reply:
I agree it's pretty hard to judge :D